test/storage/file.js

```diff
@@ -281,6 +281,126 @@ describe('File', function() {
     });
   });
 
+  describe('move', function() {
+    it('should throw if no destination is provided', function() {
+      assert.throws(function() {
+        file.move();
+      }, /should have a name/);
+    });
+
+    describe('destination types', function() {
```

Comment:
@stephenplusplus Good catch on the test style. 

@mziccard I fumble over the style we use sometimes as well. The stubs can really convolute things, but I do feel it pays off in the end. I find it really helps to look at the existing tests and copy the style used within. Hope that helps :)